### PR TITLE
Restaurant Website

### PR DESCRIPTION
**Pull request by MarkoNS1990**
🟢 In this project I built a simple Restaurant website. Using DOM manipulations and JavaScript alone to render whole content. 🟢

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,10 @@
   },
   "homepage": "https://github.com/MarkoNS1990/restaurant#readme",
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.27.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.23.3",
     "webpack": "^5.38.1",
     "webpack-cli": "^4.7.0"
   }
```
Comment (teekaytech) [inline package.json:28]: **Required**
Necessary run common commands like <code>build</code>, <code>development</code> should be in <code>package.json</code> inside the `scripts` block. Here is a basic example (<a href="https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png">screenshot</a> from <a href="https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/" rel="nofollow noreferrer noopener" target="_blank">Pathwright</a>)</li>
</ul>
<pre class="code highlight js-syntax-highlight language-json white" v-pre="true" lang="json"><code><span id="LC1" class="line" lang="json"><span class="nl">"scripts"</span><span class="p">:</span><span class="w"> </span><span class="p">{</span></span>
<span id="LC2" class="line" lang="json"><span class="w">    </span><span class="nl">"dev"</span><span class="p">:</span><span class="w"> </span><span class="s2">"webpack --mode development"</span><span class="p">,</span></span>
<span id="LC3" class="line" lang="json"><span class="w">    </span><span class="nl">"build"</span><span class="p">:</span><span class="w"> </span><span class="s2">"webpack --mode production"</span><span class="p">,</span></span>
<span id="LC4" class="line" lang="json"><span class="w">    </span><span class="nl">"start"</span><span class="p">:</span><span class="w"> </span><span class="s2">"webpack-dev-server --mode development --open"</span><span class="w"> </span><span class="err">//</span><span class="w"> </span><span class="err">optional</span></span>
<span id="LC5" class="line" lang="json"><span class="w">  </span><span class="p">}</span><span class="err">,</span>

File: src/index.js [hunk 2]
```diff
@@ -1 +1,32 @@
...
+  menuDiv.style.display = 'block';
+  contactDiv.style.display = 'none';
+});
+
+// Show only contact page
+linkContact.addEventListener('click', () => {
+  homeDiv.style.display = 'none';
+  menuDiv.style.display = 'none';
+  contactDiv.style.display = 'block';
+});
```
Comment (teekaytech) [inline src/index.js]: **Optional**
There seems to be a pattern here. I think there should be a way to refactor this code such that your code will be a bit DRY. Can you think of any way? For instance, what if you move lines 29-31 to a separate function, then you accept their values as the function arguments, what did you think?

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @MarkoNS1990 :wave:

__You've done really well so far. Here're the major highlights of this PR:__

- You've produced a working project with no major errors reported on the console :+1: 
- All linter checks are running/passing successfully :+1: 

__However, some changes that you are required to make before moving on to the next task. Please check my comments below and fix them as appropriate.__

### **Required Changes**

Check the comments under the review.

### **Optional Changes**
_Every comment with the **OPTIONAL** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- checking your markdown syntax. I can see that you tried to use table to represent the project screenshots on your Readme. This is cool, but it looks like you got the syntax wrong. [This article](https://guides.github.com/features/mastering-markdown/) might help.


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can see your comments and respond._

_**Please make the required changes and submit another review request when you are done.**_

* * *
_Regards,_
**Taofeek Abiodun Olalere**
__Cheers and happy coding :smiley:__

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Pull request merged**